src/components/groups/group.middlewares.ts

```diff
@@ -177,7 +208,10 @@ export const checkValidMaxAttendeeLimit = asyncWrapper(
 
 export const checkConflicts = asyncWrapper(
   async (req: Request, res: Response, next: NextFunction) => {
-    const group = req.body as Group
+    const { type, ...group } = req.body as Group & { type: string }
```

Comment:
Just a comment, you don't have to change anything about this:

I was thinking about maybe this `& { type: string }` type concatenating is weird, but I can see, you did not want to include type as a field in the Group entity. I don't think that this would cause any issue in the future, as there won't be any more enhancement needing this type property, so I don't mind leaving this type field out of the Group entity. It's good as is :+1: